python/cuml/test/test_pca.py

```diff
@@ -98,3 +108,5 @@ def test_pca_inverse_transform(datatype, input_type):
 
     assert array_equal(input_gdf, gdf,
                        1e-3, with_sign=True)
+    if stream is not None:
```

Comment:
Don't we need to sync in any case? Also is the sync not needed before line 109?

Reply:
Done